content/posts/getting-started/github-pages/index.fr.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Déployer dans Github Pages"
-date: 2023-11-11T22:00:20+02:00
+date: 2020-06-08T22:00:20+06:00
```

Comment:
Why change the timestamp? To change the order of post cards? By default, it puts the latest post first. If we want a custom order (probably by weight), we can introduce a configurable option for that. Changing post timestamp to order post is not a good solution.

Reply:
You're right @hossainemruz. i make rollback in the timestamp to retablish initial timestamp from English Guide because I would keep the same order of post cards between English Guide and French Guide. It's "Nice to have".

Actually, if it's not possible to force the same order post cards, then I'm giving up. On the other hand, we can make an issue about custom order. It's an good idea.

For me, you can merge nervertheless.